bin/alphanet/Cargo.toml

```diff
@@ -30,7 +30,7 @@ default = ["jemalloc"]
 
 asm-keccak = ["reth-optimism-cli/asm-keccak"]
 
-jemalloc = ["dep:tikv-jemallocator"]
+jemalloc = ["dep:tikv-jemallocator", "reth-cli-util/jemalloc"]
```

Comment:
should we not remove the `dep:tikv-jemallocator`? it's pulled in by `reth-cli-util/jemalloc`

Reply:
ah yeah, fixed